src/Analysis/Ast/Impl/Analyzer/Symbols/ClassEvaluator.cs

```diff
@@ -120,6 +120,8 @@ internal sealed class ClassEvaluator : MemberEvaluator {
                 var m = Eval.GetValueFromExpression(expr);
 
                 switch (m) {
+                    // Allow unknown members 
+                    case IMember member when member.GetPythonType() == Eval.UnknownType:
```

Comment:
Why `when member.GetPythonType() == Eval.UnknownType` over `when member.IsUnknown()`?

Reply:
In fact, `m` is already an `IMember`, so you can just do `if (m.IsUnknown())` before the `switch`.

Reply:
Ah man, I forgot about that extension method.